lxd/network/driver_physical.go

```diff
@@ -219,6 +219,12 @@ func (n *physical) Validate(config map[string]string) error {
 		return err
 	}
 
+	// Validate IP routes.
```

Comment:
The commit message:

> Check that ipv4.routes and ipv6.routes contain the routes for existing OVN network forwards and load balancers.

Would have made a better comment here than "Validate IP routes" which is clear from the function name ;)